config/initializers/mime_types.rb

```diff
@@ -1 +1,7 @@
 Mime::Type.register 'application/secevent+jwt', :secevent_jwt
+Mime::Type.register "gzip/json", :gzip
```

Comment:
can the [`Content-Encoding` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding) cover this? I would not expect us to need to make a custom GZIP + JSON combo `Content-Type`

Reply:
It doesn't unfortunately, I couldn't find anything in Rails to support it in a straightforward manner

We may want to take a different strategy for this anyway, as I suspect we'll want to limit the automatic unzipping to the specific route and only attempt it when it is authorized.